include/lsqecc/ls_instructions/ls_instructions.hpp

```diff
@@ -65,7 +65,7 @@ struct BellPairInit {
     PlaceNexTo loc2;
 
     std::optional<std::vector<LocalInstruction::LocalLSInstruction>> local_instructions;
```

Comment:
Just a general comment. Missed it last time, but are you using the optional to distinguish no vector from an empty vector in some way?

Reply:
I am using the optional since it is only needed if compiling to local instructions (however I haven't yet pushed the commit that allows there to be a difference)